app/views/mailing_list/steps/completed/shared/_book_a_callback.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2 class="heading-m">Have a question?</h2>
+
+<p>
+  Contact our helpful team of experts who can answer questions about
+  qualifications, routes into teaching or your application.
+</p>
+
+<div class="mailing-list__actions">
+  <% if callback_available? %>
+    <%= link_to("Book a callback", callbacks_steps_path, class: "button") %>
+    <div>
+      <small>or call us now:</small>
+      <p class="telephone">0800 389 2500</p>
+    </div>
+  <% else %>
+    <div>
```

Comment:
I wonder whether we should bring this 'Call us now' section below the Book a callback button to make it more prominent (we'd rather people call us now) - and then hide it when it is out of hours in the same way we do with the chat button?

However, I'd suggest we consider this on a separate ticket (if it has any merit!) rather than delay this mapping PR - let's discuss @philspray @ethax-ross